src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/testhosts/AuthenticationResourceHelper.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.ServiceModel;
+
+namespace WcfService
+{
+    public static class AuthenticationResourceHelper
```

Comment:
Not required for checkin, but we probably need to make a sweep and remove "Resource" from names.  That's a hold-over from Bridge naming